imap_processing/codice/generate_ccsds_header_xml.py

```diff
@@ -0,0 +1,121 @@
+""" All L0 data is written with a CCSDS header. This script creates the CCSDS header
+    parameters and writes them to an XML file. The XML file is then used to create
+    the CCSDS header parameters in the XTCE file.
+    """
+
+import xml.etree.ElementTree as ET
+
+ET.register_namespace("xtce", "http://www.omg.org/space")
+
+# Create the root element
+
+root = ET.Element("{http://www.omg.org/space}SpaceSystem")
+root.attrib["name"] = "CCSDS_Headers"
+
+# Create the Header element and its attributes
+header = ET.SubElement(root, "{http://www.omg.org/space}Header")
+header.attrib["date"] = "2023"
+header.attrib["version"] = "1.0"
+header.attrib["author"] = "IMAP SDC"
+
+# Create the TelemetryMetaData element
+telemetry_metadata = ET.SubElement(root, "{http://www.omg.org/space}TelemetryMetaData")
+
+# Create the ParameterTypeSet element
+parameter_type_set = ET.SubElement(
+    telemetry_metadata, "{http://www.omg.org/space}ParameterTypeSet"
+)
+
+# Create integer parameter types
+integer_sizes = [1, 2, 3, 11, 14, 16, 32]
+for size in integer_sizes:
+    parameter_type = ET.SubElement(
+        parameter_type_set, "{http://www.omg.org/space}IntegerParameterType"
+    )
+    parameter_type.attrib["name"] = f"uint{size}"
+    parameter_type.attrib["signed"] = "false"
+
+    encoding = ET.SubElement(
+        parameter_type, "{http://www.omg.org/space}IntegerDataEncoding"
+    )
+    encoding.attrib["sizeInBits"] = str(size)
+    encoding.attrib["encoding"] = "unsigned"
+
+    unit_set = ET.SubElement(parameter_type, "{http://www.omg.org/space}UnitSet")
+
+# Create the ParameterSet element
+parameter_set = ET.SubElement(
+    telemetry_metadata, "{http://www.omg.org/space}ParameterSet"
+)
+
+# Create CCSDS Header parameters
+ccsds_parameters = [
```

Comment:
These parameters should also come from the new CcsdsHeaderXtceGenerator class you made, to avoid duplicating things. Otherwise, this script looks great for what I was thinking!